src/tello/tello.py

```diff
@@ -488,3 +488,37 @@ def move_up(self, distance) -> str:
         """
 
         return self.move('up', distance)
+
+    def go_xyz_speed(self, x: int, y: int, z: int, speed: int):
+        """Fly to x y z relative to the current position.
+        Speed defines the traveling speed in cm/s.
+        Arguments:
+            x: 20-500
+            y: 20-500
+            z: 20-500
+            speed: 10-100
+        """
+        x = self._check_limits(x, 20, 500)
+        y = self._check_limits(y, 20, 500)
```

Comment:
what are 20 and 500? Magic numbers :D

Reply:
minimum and maximum shift in one operation, I must also add a range -500 to -20 (backward movement)